src/utils/inputValidations.js

```diff
@@ -23,3 +23,5 @@ export const numericalFourDigits = errorMessage => pattern(numericalFourDigitsRe
 export const numericalTenDigits = errorMessage => pattern(numericalTenDigitsRegex, errorMessage);
 
 export const numericalDigits = errorMessage => pattern(numericalDigitsRegex, errorMessage);
+
+export const validatePassCoincidence = (pass, confirmPass) => String(pass) === String(confirmPass);
```

Comment:
I think is unnecessary 
```suggestion
export const validatePassCoincidence = (pass, confirmPass) => pass === confirmPass;
```